controllers/volumegroup_controller.go

```diff
@@ -102,15 +102,15 @@ func (r *VolumeGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 	} else {
-		if utils.Contains(instance.GetFinalizers(), utils.VGFinalizer) {
+		if utils.Contains(instance.GetFinalizers(), utils.VGFinalizer) && !utils.IsContainOtherFinalizers(instance, logger) {
```

Comment:
not a blocker but I think it can be exported to a function like `isVGHasOnlyVGFInalizers` or something like this, it can be cleaner since we are doing it in other plcaces